resources/modtemplate/main.lua

```diff
@@ -373,7 +373,7 @@ function BasementRenovator.RenderDoorSlots(room, doorSlots, enterSlot)
     end
 end
 
-BasementRenovator.mod:AddCallback(ModCallbacks.MC_POST_NEW_ROOM, function()
+BasementRenovator.mod:AddPriorityCallback(ModCallbacks.MC_POST_NEW_ROOM, CallbackPriority.IMPORTANT, function()
```

Comment:
Since BR is technically supposed to work with the different versions of the game (although it's best effort right now) when using rep only APIs I've been gating them behind checks: https://github.com/Basement-Renovator/basement-renovator/commit/b4ffe884efb48863a99dca02b0b35767094ef654#diff-09679ecfabcf35045938309e4229f4c96e65f8392a5b1b6f643aeb53ee16ecf4R211

Could you create some kind of wrapper for AddPriorityCallback that just uses AddCallback when it's not available?